fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -134,25 +157,13 @@ protected void onCreate(Bundle savedInstanceState) {
                 setupLaunchFragment();
               }
             });
-
-    // Subscribe to "open settings" event
-    mViewModel
-        .getOpenSettingsEvent()
-        .observe(
-            this,
-            booleanEvent -> {
-              Boolean event = booleanEvent.getContentIfNotHandled();
-              if (event != null) {
-                setupSettingsActivity();
-              }
-            });
-
-    subscribeWalletEvents();
-    subscribeSocialMediaEvent();
   }
 
   private void subscribeWalletEvents() {
 
+    MenuItem connectItem = navbar.getMenu().getItem(1);
+    MenuItem launchItem = navbar.getMenu().getItem(2);
```

Comment:
This is really clean but I think that defining constants or an enum instead of having getItem(1) getItem(2) is even more clean and more understandable in my opinion